packages/protocol/contracts/Staking.sol

```diff
@@ -502,6 +479,7 @@ contract Staking is ERC900, IStakingLocking, IsContract, TimeHelpers {
     }
 
     /**
+      // Note: not sure I understand this comment, since we don't use msg.sender internally
```

Comment:
Clarifying a bit, but if you look at all the calls of the function, `msg.sender` is always `_user` or `_lockManager` (or it should be!)